services/ingestion/read_from_channel.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/techrail/bark/models"
 )
 
+// InsertSingle and InsertMultiple functions have a common functionality - pushing the logs into the LogChannel.
+// The channel capacities are checked beforehand. In case, the channels are full, that message is conveyed.
+// Each log entry's values are validated and then sent to the LogChannel.
```

Comment:
We should still write a single line comment for both functions, for the sake of uniformity across the codebase.

Reply:
Added comments for both functions.